parsl/tests/test_error_handling/test_resource_spec.py

```diff
@@ -11,19 +13,42 @@ def double(x, parsl_resource_specification={}):
 
 
 def test_resource(n=2):
-    spec = {'cores': 2, 'memory': '1GiB'}
+    executors = parsl.dfk().executors
+    executor = None
+    for label in executors:
+        if label != 'data_manager':
+            executor = executors[label]
+            break
+
+    # Specify incorrect number of resources
+    spec = {'cores': 2, 'memory': 1000}
+    fut = double(n, parsl_resource_specification=spec)
+    try:
+        fut.result()
+    except UnsupportedFeatureError:
+        assert not isinstance(executor, WorkQueueExecutor)
+    except Exception as e:
+        assert isinstance(e, ExecutorError)
+
+    # Specify resources with wrong types
+    # cpus is incorrect
```

Comment:
for clarity consider changing to:

Resource specification has incorrect keys: `cpus` should be `cores`